scipy/stats/tests/test_distributions.py

```diff
@@ -5297,6 +5297,19 @@ def test_sf_isf(self):
         x1 = stats.nakagami.isf(sf, nu)
         assert_allclose(x1, x0, rtol=1e-13)
 
+    @pytest.mark.parametrize('nu', [1.6, 2.5, 3.9])
+    @pytest.mark.parametrize('loc', [25.0, 10, 35])
+    @pytest.mark.parametrize('scale', [13, 5, 20])
+    def test_fit(self, nu, loc, scale):
+        # The first tuple of the parameters' values is discussed in gh-10908
+        samples = stats.nakagami.rvs(size=100, nu=nu, loc=loc,
+                                     scale=scale, random_state=1337)
+        nu_est, loc_est, scale_est = stats.nakagami.fit(samples)
+        assert_almost_equal(nu_est, nu, decimal=0)
```

Comment:
The tolerance is very loose here. Let me think about this change some more before I suggest a test.
(BTW, the documentation for assert_almost_equal now mentioned that assert_allclose is preferred.)

Reply:
Yes, that's because I use a small value for `size`. As said in the PR description, maybe implementing analytical ML estimators for Nakagami would be preferred. I am reading about other issues and PRs about scipy.stats to know what the current direction is regarding MLE, especially analytical solutions or not.

Reply:
Yes, analytical fits are preferred, but sometimes they are a lot more work.
I am going to guess that the analytical fit for the location is actually the minimum of the data rather than the mean, btw. That's what I'm going to look into.

Reply:
You can take a look at some of the other distributions with recently overridden `fit` methods in gh-11782 (more specifically [here](https://github.com/scipy/scipy/issues/11782#issuecomment-633188975)).

Reply:
Thanks, this is useful information to me!

Reply:
I think this PR can be seen as a hot-fix for now. I can have a look at the MLE for the general form of Nakagami in a second PR: let me know. 🙂

Reply:
I think that the analytical MLE for scale, for SciPy's parameterization, is
![image](https://user-images.githubusercontent.com/6570539/104857527-f2348400-58cd-11eb-80aa-c8eeafa5908b.png)

Reply:
Also, the mean isn't a good guess for the location, because the support of the distribution tells us that `xi - loc >= 0` for all xi, so `loc <= min(xi)`. I don't the equality provides the exact MLE, but I think `loc = np.min(xi)` and scale as provided above would be theoretically justified guesses.

Reply:
Thanks for the details. Do you have a reference for those results? This way I could cite them in the implementation.

Reply:
Please do provide the nasty equation so that I can see how it compares.

Reply:
OK, I'll include it in a writeup.